.eslintrc

```diff
@@ -1,176 +1,176 @@
 {
```

Comment:
ES6 features:

No "classes" or "superInFunctions" -- everything else is great. :+1: 

[The Two Pillars of JavaScript](https://medium.com/javascript-scene/the-two-pillars-of-javascript-ee6f3281e7f3)
[How to Fix the ES6 Class Keyword](https://medium.com/javascript-scene/how-to-fix-the-es6-class-keyword-2d42bb3f4caf)
[Common Misconceptions About Inheritance in JavaScript](https://medium.com/javascript-scene/common-misconceptions-about-inheritance-in-javascript-d5d9bab29b0a)